controllers/druid/testdata/broker-statefulset.yaml

```diff
@@ -32,8 +32,10 @@ spec:
       affinity: {}
       containers:
       - command:
-        - bin/run-druid.sh
-        - broker
+        - sh
+        args:
+        - "-c for i in /opt/druid/conf/druid/cluster/env/*.env; do source $i; done && /druid.sh"
```

Comment:
shouldn't it be bin/druid.sh?

Reply:
i think it's `/druid.sh`, according to our current statefulsets